src/main/java/jenkins/plugins/office365connector/CardBuilder.java

```diff
@@ -194,19 +194,14 @@ public Card createBuildMessageCard(StepParameters stepParameters) {
     }
 
     private void addScmDetails() {
-        Set<User> users;
-        List<ChangeLogSet<ChangeLogSet.Entry>> sets;
-
-        try {
-            users = (Set<User>) run.getClass().getMethod("getCulprits").invoke(run);
-            sets = (List<ChangeLogSet<ChangeLogSet.Entry>>) run.getClass().getMethod("getChangeSets").invoke(run);
-        } catch (IllegalAccessException | InvocationTargetException | NoSuchMethodException e) {
-            users = Collections.emptySet();
-            sets = Collections.emptyList();
+        if (!(run instanceof AbstractBuild)) {
```

Comment:
This breaks support for Jenkins pipeline jobs.
Please add a test for Jenkins pipeline job.